dags/aggregates_python_helpers.py

```diff
@@ -72,14 +69,24 @@ def create_roman_set():
         roman_set.add(roman_number)
     return roman_set
 
-def load_data_from_csv_to_db(file_path, db_params, ti=None, **kwargs):
+def load_data_from_csv_to_db(file_path,  ti=None, **kwargs):
```

Comment:
the name of the function suggerst that we could load 'some' data to database. the function is parametrixed by file_path.
I think it could be more precise and write that you are "load_permissionss_to_bq" and parametrize this by file_path - if this is needed, bigquery table parametes(destination parameters)